src/overview/pricing.md

```diff
@@ -1,26 +1,26 @@
 # Platform.sh pricing
 
-All Platform.sh plans include the following:
+All Platform.sh plans include:
 
-* Four Environments (3 for staging/development 1 for the live site).
-* One Developer License
+* four Environments (3 for staging/development, 1 for the live site).
+* one Developer license
 * 5GB of Storage per environment
-* Multiple Backend services (MySQL, PostgreSQL, Elasticsearch, Redis, Solr..)
-* Support
+* multiple Backend services (MySQL, PostgreSQL, Elasticsearch, Redis, Solr..)
+* support
 
-You can switch between plans (downgrade or upgrade) freely but note that
+You can switch between plans (downgrade or upgrade) freely, but note that
 reducing storage is currently not supported for technical reasons. If you need
-to reduce storage please create a support ticket. You will always be billed the
+to reduce storage, please create a support ticket. You will always be billed the
 prorated rate of your plan over the period it was used.
 
 You may cancel your plan at any time and you will only be billed for the actual
 period used.
```

Comment:
Ibid.  Single line while we're at it.

Reply:
(And in the rest of the file.  I won't note each place it's needed here.)

Reply:
Done.